### ISSUE TITLE
Why use a convolution kernel of size 16 in this paper? 

### ISSUE THREAD

**Issue by CN-zdy** [antonior92/automatic-ecg-diagnosis#11]
Hi, this is a very interesting paper documenting ability of a deep neural network (DNN) to identify abnormalities in a standard 12-lead ECG recordings. I have some confusion after reading the paper.

Q1: The convolution kernel of size 3 is widely used in the field of CV. Why use a convolution kernel of size 16 in this paper?

Q2: Other sizes of convolution kernels are listed in the paper. Why are the results not attached?

"The convolutional layers have filter length 16, starting with 4096 samples and 64 filters for the first layer and residual block and increasing the number of filters by 64 every second residual block and subsampling by a factor of 4 every residual block. Max Pooling51 and convolutional layers with filter length 1 (1x1 Conv) are included in the skip connections to make the dimensions match those from the signals in the main branch."

"The hyperparameters were chosen among the following options: residual neural networks with {2, 4, 8, 16} residual blocks, kernel size {8, 16, 32}, batch size {16, 32, 64}, initial learning rate {0.01, 0.001, 0.0001}, optimization algorithms {SGD, ADAM}, activation functions {ReLU, ELU}, dropout rate {0, 0.5, 0.8}, number of epochs without improvement in plateaus between 5 and 10, that would result in a reduction in the learning rate between 0.1 and 0.5. "

**Comment by antonior92** [comment]
Hi @CN-zdy, sorry for the long time to respond.

Q1: Convolutions with size k=3  are, indeed, widely used in CV.  Note, however, that the number of parameters in the convolutional layer for 2D convolution scale with k^2. For 1D convolutions, it scales with k. Thinking this way it is easier to see that the number of parameters for 1D convolutional layer with k=16 is comparable with 2D convolutions with more or less standard kernel size in CV problems. Moreover, during the experiments, we obtained the best results for this kernel size.

Q2: I am sorry about that. This was our first work in deep learning and back then we had only a couple of GPUs and the training took ~1week. So a lot of the hyperparameter search happened in parallel with the problem development and the construction of the training set. So I would say that some of obtained results were obtained in slightly different settings. While we used the insight from it to choose the final model, it is hard to directly compare them...
